app/src/main/java/com/ex/org/exceptionstest/com/ex/lib/ExceptionReporter.java

```diff
@@ -36,6 +34,8 @@ private ExceptionReporter(Context ctx){
 
     };
 
+    //ERIK - is there any specific reason why you require this to be an Application and not just a Context?
```

Comment:
yeah i thought about it, wasn't sure if it's any better to call context.getApplicaitonContext and left it as is

Reply:
Yeah, I think just using context is always better unless you got a very specific requirement.

Reply:
👍